File: FPNg-API/FPNg.API.Data/Domain/Ledger.cs
using System;

namespace FPNg.API.Data.Domain
{
    public class Ledger
    {
        public int? RollupKey { get; set; }
        public int? Year { get; set; }
        public DateTime? WDate { get; set; }
        public double? CreditSummary { get; set; }
        public double? DebitSummary { get; set; }
        public double? Net { get; set; }
        public double? RunningTotal { get; set; }
        public DateTime? OccurrenceDate { get; set; }
        public int? ItemType { get; set; }
        public string PeriodName { get; set; }
        public string Name { get; set; }
        public double? Amount { get; set; }
    }
}

File: FPNg-UI/src/app/feature/test/common/models/ledger.ts
export interface ILedgerVM {
  rollupKey: number;
  year?: number;
  wDate?: Date;
  creditSummary?: number;
  debitSummary?: number;
  net?: number;
  runningTotal?: number;
  itemCount?: number;
  items?: IItemVM[];
}
export interface IItemVM {
  itemKey: number;
  occurrenceDate?: Date;
  itemType?: number;
  periodName?: string;
  name?: string;
  amount?: number;
}

export interface ILedger {
  rollupKey: number;
  year?: number;
  wDate?: Date;
  creditSummary?: number;
  debitSummary?: number;
  net?: number;
  runningTotal?: number;
  occurrenceDate?: Date;
  itemType?: number;
  periodName?: string;
  name?: string;
  amount?: number;
}

